documentation/pdb_support_guide.markdown

```diff
@@ -31,15 +31,78 @@ The terminus resides on the Puppet master and redirects agent data to
 PDB in the form of "commands". PDB has four commands, as described in the
 [commands documentation][commands].
 
-### Queue
+### Message Queue
 
-The commands sent from the terminus to PDB are likely to be deferred
-to a filesystem queue for later processing, at which point they will
-be submitted to the database.  Commands may not always be processed in
-the order received, but will ususally be handled in roughly the order
-submitted.  Although "stale" commands might be ignored.  For example,
-older fact sets or catalogs that are still in the queue might be
-dropped when newer ones are received for the same certname.
+The commands sent from the terminus to PDB are likely to be (currently
+always) deferred to a filesystem queue for later processing, at which
+point they will be submitted to the database.  Commands may not always
+be processed in the order received, but will ususally be handled in
+roughly the order submitted, though "stale" commands might be ignored.
```

Comment:
There's a lot of grey area here, and I'm not sure why. These docs are tied to the version of PDB that we are about to release. In that version, they will always go to disk, and we'll not ignore stale commands. The not processed in order stuff, though technically correct, really it can only happen if we are starting up with a backed up queue. Although it might be worth mentioning at some point, the first few sentences doesn't seem like the right place to me.

On the things that we are planning to change in the near future, it might be worth noting at the end or just leaving out entirely (what if we never get around to it?)